index.html

```diff
@@ -13,7 +13,7 @@
                 <img class="logo" src="logo1.png">
             <div class="forminput">
                 <label class="l"><img src="username.png"></label>
-                <input type="text" placeholder="Name">
+                <input type="text" placeholder="Fixoc">
             </div>
             <div class="forminput">
                 <label><img src="password.png"></label>
```

Comment:
alignment issues in the design w.r.t provided design, and indentation issues at code level. Please fix.

Reply:
corrected